canonicalwebteam/discourse_docs/app.py

```diff
@@ -51,6 +51,9 @@ def document_view(path=""):
             if path == "/":
                 document = index
             else:
+                if path in index["redirect_map"]:
+                    return flask.redirect(index["redirect_map"][path])
```

Comment:
If this redirects here, I assume it ends up in this function again? Which is how it then reaches line https://github.com/canonical-web-and-design/canonicalwebteam.discourse-docs/pull/6/files#diff-c8a4d7795295f059092ac2ae948cbea3R92

Reply:
yes, it will end up here again, with a path that presumably isn't in the redirect map.

What's at line 92?

Reply:
The warnings. Was just wondering how it will end up there.

Reply:
Oh yeah the warnings are only added to successful 200 documentation pages. What it's saying is, "for this (200) documentation page you just requested, I just want to warn you some of the parsing of the navigation config failed".

As in, whether you go to `/a-redirect` which redirects you to `/a-real-page`, or whether you go to `/a-real-page` directly, when you get there you'll see these warning headers appear to warn you that some items in the redirect config are misconfigured and were ignored.

Reply:
I could add these headers to more of the responses, like maybe the 404 response. It's just that I couldn't find a way to easily add it to all responses without just copying and pasting the code =)